lib/util/mp4_generator.js

```diff
@@ -198,29 +175,31 @@ shaka.util.Mp4Generator = class {
   /**
    * Generate a MDIA box
    *
+   * @param {shaka.util.Mp4Generator.StreamInfo} streamInfo
    * @return {!Uint8Array}
    * @private
    */
-  mdia_() {
+  mdia_(streamInfo) {
     const Mp4Generator = shaka.util.Mp4Generator;
-    return Mp4Generator.box(
-        'mdia', this.mdhd_(), this.hdlr_(), this.minf_());
+    return Mp4Generator.box( 'mdia', this.mdhd_(streamInfo),
```

Comment:
It looks like you added extra whitespace before the `'mdia'`?

Reply:
Fixed!